mlflow/server/js/src/gateway/components/model-configuration/index.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Model Configuration Components
```

Comment:
can we remove this file? We typically don't have index.ts just for re-exporting.